Sources/Basic/PackageDir.swift

```diff
@@ -0,0 +1,28 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import Utility
+
+public func isSafeToRemove(_ item: String) -> Bool {
```

Comment:
For now, I would recommend putting it with `Utility/Git.swift`. I'll clean up a final location as part of my source management work.